### PR TITLE
docs: require screenshots for interface changes

### PR DESCRIPTION
**Pull request by ascorbic**
## What does this PR do?

Requires screenshots for issues that refer to the interface and PRs that change the UI.

Documents how agents and contributors can upload local screenshots with GitHub CLI 2.99.0 or later, including repeatable `--attach` flags, alt text, in-body placement, and repository write-access requirements. Updates the PR template, bug-report form, and triage checklist so the requirement appears where reports and reviews are created.

Related issue: Not applicable.

## Type of change

- [ ] Bug fix
- [ ] Feature (requires [maintainer-approved Discussion](https://github.com/emdash-cms/emdash/discussions/categories/ideas))
- [ ] Refactor (no behavior change)
- [ ] Translation
- [x] Documentation
- [ ] Performance improvement
- [ ] Tests
- [ ] Chore (dependencies, CI, tooling)

## Checklist

- [x] I have read [CONTRIBUTING.md](https://github.com/emdash-cms/emdash/blob/main/CONTRIBUTING.md)
- [x] `pnpm typecheck` passes
- [x] `pnpm lint` passes
- [ ] `pnpm test` passes (not applicable; this PR changes documentation and GitHub templates only)
- [ ] `pnpm format` has been run (changed files pass a targeted Prettier check)
- [ ] I have added/updated tests for my changes (not applicable; there is no executable behavior)
- [ ] User-visible strings in the admin UI are [wrapped for translation](https://github.com/emdash-cms/emdash/blob/main/CONTRIBUTING.md#internationalization-i18n) (not applicable; there are no admin UI changes)
- [ ] I have added and reviewed the user-facing [changeset](https://github.com/emdash-cms/emdash/blob/main/.changeset/README.md) (not applicable; no published package changes)
- [ ] New features link to an approved Discussion (not applicable; this is contributor-policy documentation)
- [ ] I have included screenshots below if this PR changes the UI (not applicable; this PR does not change the UI)

## AI-generated code disclosure

- [x] This PR includes AI-generated code — model/tool: Codex (GPT-5)

## Screenshots / test output

Not applicable: this PR does not change the UI.

Verified with:

- `pnpm build`
- `pnpm typecheck`
- `pnpm lint:quick`
- `pnpm lint:json` — 0 diagnostics
- `pnpm --dir docs build`
- `pnpm exec prettier --check AGENTS.md CONTRIBUTING.md TRIAGE.md .github/PULL_REQUEST_TEMPLATE.md .github/ISSUE_TEMPLATE/bug_report.yml`
- YAML parse of `.github/ISSUE_TEMPLATE/bug_report.yml`
- `git diff --check`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -153,12 +153,29 @@ AI-assisted contributions are welcome and held to the same quality bar as any ot
 - AI-generated PRs must pass CI, follow project patterns, and include tests.
 - Check the PR template's AI disclosure box and name the model/tool (e.g. Claude Opus 4.7, GPT-5.5, Cursor + Sonnet 4.6). This isn't punitive -- it helps reviewers focus on edge cases that AI tools tend to miss and run the review pass with a different model family.
 
+### Interface screenshots
+
+An issue that refers to the interface must include a screenshot showing the reported state. A PR that changes the UI must include screenshots of the rendered result. Include before-and-after images when the result alone does not make the change clear. Describe the behavior in text as well, and use alt text that identifies the screen and relevant state.
+
+In the GitHub web interface, drag or paste the images into the issue or PR body. From GitHub CLI 2.99.0 or later, use the repeatable `--attach` flag with issue and PR create, edit, or comment commands. Image uploads require write access to the repository.
```
[the remark this answers is not in the document]
Comment (ascorbic) [reply CONTRIBUTING.md]: Fixed in b9ba91ade: this now says that CLI `--attach` uploads require write access, while web interface uploads do not.

File: AGENTS.md [hunk 2]
```diff
@@ -36,6 +36,21 @@ A changeset is user-facing documentation that lands verbatim in a package CHANGE
...
+	--attach './interface-error.png#The settings screen showing the validation error'
+
+gh pr create --body-file /tmp/emdash-pr.md \
+	--attach './before.png#Settings screen before the change' \
+	--attach './after.png#Settings screen after the change'
+```
+
+To place an image at a specific point in the body, add `![descriptive alt text](./after.png)` to the body file and pass `--attach ./after.png`; `gh` replaces the local path with the uploaded asset URL. An attachment that is not referenced in the body is appended. Image uploads require repository write access. See [CONTRIBUTING.md § Interface screenshots](CONTRIBUTING.md#interface-screenshots).
+
 ## Architecture
```
[the remark this answers is not in the document]
Comment (ascorbic) [reply AGENTS.md:55]: Fixed in b9ba91ade: the write-access requirement is now explicitly scoped to CLI `--attach` uploads.

File: AGENTS.md [hunk 3]
```diff
@@ -36,6 +36,21 @@ A changeset is user-facing documentation that lands verbatim in a package CHANGE
 
 When opening a PR with `gh`/the API, copy `.github/PULL_REQUEST_TEMPLATE.md` into the body and fill every section -- the GitHub UI injects it automatically but the CLI does not, and PRs missing it are auto-closed. Check the AI-generated code disclosure box and name the model. Tick checklist items only for what you actually verified; for test-only/docs/CI PRs, note why changeset/i18n/Discussion items are n/a.
 
+Issues that refer to the interface must include a screenshot that shows the reported state. PRs that change the UI must include screenshots of the rendered result; include before-and-after images when the change is not clear from the result alone. Keep the behavior described in text and give every image useful alt text.
+
+Agents can attach local images with GitHub CLI 2.99.0 or later. The `--attach` flag is repeatable on `gh issue create|edit|comment` and `gh pr create|edit|comment`. For example:
+
+```bash
+gh issue create --body-file /tmp/emdash-issue.md \
```
[the remark this answers is not in the document]
Comment (ascorbic) [reply AGENTS.md:44]: Confirmed against GitHub’s September 1 announcement and linked documentation: `--attach` requires GitHub CLI 2.99.0 or later, is repeatable, and supports issue/PR create, edit, and comment commands. No wording change needed. https://github.blog/changelog/2026-09-01-github-cli-media-in-issues-pull-requests-and-comments/
